nuget.config

```diff
@@ -2,8 +2,6 @@
 <configuration>
   <packageSources>
     <clear/>
-    <add key="durabletask" value="https://pkgs.dev.azure.com/durabletaskframework/734e7913-2fab-4624-a174-bc57fe96f95d/_packaging/durabletask/nuget/v3/index.json" />
-    <add key="AzureFunctionsTempStaging" value="https://pkgs.dev.azure.com/azfunc/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsTempStaging/nuget/v3/index.json" />
```

Comment:
we remove this source for durable extension, I guess it should be fine to be removed here too?